drivers/phy/marvell/phy-mvebu-cp110-comphy.c

```diff
@@ -829,7 +848,18 @@ static int mvebu_comphy_power_on(struct phy *phy)
 
 try_legacy:
 	/* Fallback to Linux's implementation */
-	return mvebu_comphy_power_on_legacy(phy);
+	ret = mvebu_comphy_power_on_legacy(phy);
+
+out:
+	if (!ret 
+		&& lane->mode == PHY_MODE_ETHERNET 
+		&& lane->submode == PHY_INTERFACE_MODE_10GBASER 
+		&& lane->enable_emp_quirk) {
+		dev_warn(priv->dev, "Performing emphesis workaround on PHY %d", lane->id);
```

Comment:
Typo? emphasis?

Reply:
sorry, didn't notice it on other branch PR

Reply:
NP. Corrected.